python/prairielearn.py

```diff
@@ -1776,3 +1776,7 @@ def index2key(i):
         key = chr(ord("a") + i)
 
     return key
+
+
+def is_within_limits(n: int) -> bool:
```

Comment:
Nit: this feels a bit generic when read from a callsite (which limits? is *what* within those limits?). Consider a name like `is_int_json_serializable` instead?

Reply:
Done in https://github.com/PrairieLearn/PrairieLearn/pull/6980/commits/1d2f92670d5dc5699c47a66f85c111f7f5cae091